docs/glossary.rst

```diff
@@ -693,13 +741,28 @@ Glossary of Terms
    System Integrity Management Platform
       A security framework that sits on top of :term:`RHEL` or :term:`CentOS`.
 
+   SIMP CE
+   SIMP Community Edition
+      The :term:`FOSS` version of SIMP freely available under the Apache 2.0
+      license.
+
    SIMP Compliance Engine
       A SIMP component that adds the capability to evaluate your
       :term:`Puppet` code for compliance with a policy as well as enforcing
       that the code enacts the specified policy.
 
       See: `SIMP Compliance Engine Repository <https://github.com/simp/pupmod-simp-compliance_markup>`__
 
+   SIMP Compliance Profile
+      A collection of data that maps policy directly to Puppet class and
+      defined type parameters. These profiles are used by the :term:`SIMP
+      Compliance Engine`.
+
+   SIMP EE
+   SIMP Enterprise Edition
+      A version of SIMP with commercial support by Onyx Point, Inc. that
+      provides additional capabilities beyond :term:`SIMP CE`.
```

Comment:
It might be nice if y'all could reference a document that describes those "additional capabilities".  Hell, I might even get my boss to buy a license.